code/modules/mob/living/simple_animal/abnormality/zayin/fairy_festival.dm

```diff
@@ -99,9 +99,77 @@
 		user.gib()
 	return
 
+/mob/living/simple_animal/hostile/abnormality/fairy_festival/BreachEffect(mob/living/carbon/human/user, breach_type = BREACH_NORMAL)
+	if(breach_type == BREACH_PINK)
+		SummonGuys()
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/fairy_festival/proc/SummonGuys(again = TRUE)
+	var/mob/living/simple_animal/hostile/ordeal/pink_midnight/pink = locate() in GLOB.mob_living_list
+	for(var/i = 1 to 6)
+		var/turf/target_turf = get_turf(pink ? pink : src)
+		var/mob/living/simple_animal/hostile/mini_fairy/new_fairy = new(target_turf)
+		if(pink)
+			new_fairy.faction += "pink_midnight"
+	if(again)
+		addtimer(CALLBACK(src, .proc/SummonGuys, FALSE), 20 SECONDS)
+
 /datum/reagent/abnormality/fairy_festival
 	name = "Nectar of an Unknown Flower"
 	description = "The fairies got this for you..."
 	color = "#e4d0b2"
 	health_restore = 2
 	armor_mods = list(-2, 0, 0, 0)
+
+/mob/living/simple_animal/hostile/mini_fairy
+	name = "\improper Lost Fairy"
+	desc = "They wander in search of food."
+	icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	icon_state = "fairy_bastard"
+	icon_living = "fairy_bastard"
+	maxHealth = 83
+	health = 83
+	attack_verb_continuous = "bites"
+	attack_verb_simple = "bite"
+	is_flying_animal = TRUE
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 1.2, BLACK_DAMAGE = 1.2, PALE_DAMAGE = 1.2)
+	faction = list("hostile", "fairy")
+	melee_damage_lower = 1
+	melee_damage_upper = 5
+	melee_damage_type = RED_DAMAGE
+	obj_damage = 3
+	rapid_melee = 3
+	attack_sound = 'sound/abnormalities/mountain/bite.ogg'
```

Comment:
No.

Reply:
Assuming by "No" you mean "change the sound effect"

Reply:
SFX changed to a modified version of their kill SFX from LC